application/cms/forms.py

```diff
@@ -103,13 +103,13 @@ def __init__(self, *args, **kwargs):
     type_of_statistic_id = RadioField(label="Type of statistic", coerce=int, validators=[Optional()])
 
     department_source = StringField(label="Publisher")
-    source_url = URLField(label="URL")
-    published_date = StringField(label="Publication release date")
+    source_url = URLField(label="Link")
+    published_date = StringField(label="Published on")
```

Comment:
"Publication date" seems clearer to me  - makes it clear that the box expects a date without needing additional help text.  And anyone working with statistical sources will understand the concept of a publication date.

Reply:
Please amend to:  'Source data publication date'